xml/deploy_cephadm.xml

```diff
@@ -964,5 +1040,787 @@ ses-min4
+    </para>
+<screen>
+drive_group_default_name:
+  target: *
+  data_devices:
+    drive_spec: <replaceable>DEVICE_SPECIFICATION</replaceable>
+  journal_devices:
+    drive_spec: <replaceable>DEVICE_SPECIFICATION</replaceable>
+  format: filestore
+  encryption: True
+</screen>
+   </sect3>
+   <sect3>
+    <title>Matching Disk Devices</title>
+    <para>
+     You can describe the specification using the following filters:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <para>
+       By a disk model:
+      </para>
+<screen>
+model: <replaceable>DISK_MODEL_STRING</replaceable>
+</screen>
+     </listitem>
+     <listitem>
+      <para>
+       By a disk vendor:
+      </para>
+<screen>
+vendor: <replaceable>DISK_VENDOR_STRING</replaceable>
+</screen>
+      <tip>
+       <title>Lowercase Vendor String</title>
+       <para>
+        Always lowercase the <replaceable>DISK_VENDOR_STRING</replaceable>.
+       </para>
+      </tip>
+     </listitem>
+     <listitem>
+      <para>
+       Whether a disk is rotational or not. SSDs and NVME drives are not
+       rotational.
+      </para>
+<screen>
+rotational: 0
+</screen>
+     </listitem>
+     <listitem>
+      <para>
+       Deploy a node using <emphasis>all</emphasis> available drives for OSDs:
+      </para>
+<screen>
+data_devices:
+  all: true
+</screen>
+     </listitem>
+     <listitem>
+      <para>
+       Additionally, by limiting the number of matching disks:
+      </para>
+<screen>
+limit: 10
+</screen>
+     </listitem>
+    </itemizedlist>
+   </sect3>
+   <sect3>
+    <title>Filtering Devices by Size</title>
+    <para>
+     You can filter disk devices by their size&mdash;either by an exact size,
+     or a size range. The <option>size:</option> parameter accepts arguments in
+     the following form:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <para>
+       '10G' - Includes disks of an exact size.
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       '10G:40G' - Includes disks whose size is within the range.
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       ':10G' - Includes disks less than or equal to 10&nbsp;GB in size.
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       '40G:' - Includes disks equal to or greater than 40&nbsp;GB in size.
+      </para>
+     </listitem>
+    </itemizedlist>
+    <example>
+     <title>Matching by Disk Size</title>
+<screen>
+drive_group_default:
+  target: '*'
+  data_devices:
+    size: '40TB:'
+  db_devices:
+    size: ':2TB'
+</screen>
+    </example>
+    <note>
+     <title>Quotes Required</title>
+     <para>
+      When using the ':' delimiter, you need to enclose the size in quotes,
+      otherwise the ':' sign will be interpreted as a new configuration hash.
+     </para>
+    </note>
+    <tip>
+     <title>Unit Shortcuts</title>
+     <para>
+      Instead of (G)igabytes, you can specify the sizes in (M)egabytes or
+      (T)erabytes as well.
```

Comment:
Does this mean marketing  mega/giga/tera or binary mebi/gibi/tibi?

Reply:
normal units, such as 1GB = 1024 * 1024 * 1024 bytes